arches/app/views/manifest_manager.py

```diff
@@ -112,11 +112,14 @@ def create_canvas(image_json, file_url, file_name, image_id):
         def add_canvases(manifest, canvases):
             manifest.manifest["sequences"][0]["canvases"] += canvases
 
+        def check_canvas_in_use(canvas_id):
+            return models.VwAnnotation.objects.filter(canvas__contains=canvas_id).exists()
```

Comment:
I think this is what we need, to avoid causing #10055 (sorry!)
```suggestion
            return models.VwAnnotation.objects.filter(canvas=canvas_id).exists()
```